pkg/distro/fedora/distro.go

```diff
@@ -689,6 +707,7 @@ func newDistro(version int) distro.Distro {
 			UEFIVendor: "fedora",
 		},
 		iotQcow2ImgType,
+		coreosQcow2ImgType,
```

Comment:
What about aarch64?

Reply:
I added it at first but then realised I needed to get the base container in our CI registry to test it and decided to get this open so people can review and I can add it in a follow-up.